src/components/demo/DOITooltip.css

```diff
@@ -5,7 +5,7 @@
 }
 
 .tooltip {
-    z-index: 1000;
+    z-index: -1;
```

Comment:
Hardcoding?? :eyes:

Reply:
Not really, by default its now -1, the active class makes it 1000